src/middlewares/sanitizeFields.ts

```diff
@@ -0,0 +1,38 @@
+import { NextApiHandler, NextApiRequest, NextApiResponse } from 'next';
+
+const fieldsToSanitize = ['firstName', 'lastName'];
+
+// resource: https://daringfireball.net/2010/07/improved_regex_for_matching_urls
+const removeURLsRegex =
```

Comment:
regex should also catch URLs like [phishing.com](#) (or any other TLD, without http/s prefix), which is rendered as a link on mail

Reply:
Good call. I'll update it, thanks!

Reply:
I ended up installing https://github.com/validatorjs/validator.js I think its going to give us wider coverage in this case.